sktime/classification/frequency_based/_rise.py

```diff
@@ -202,13 +204,13 @@ def fit(self, X, y):
             delayed(_parallel_build_trees)(
                 X,
                 y,
-                self.acf_lag_,
+                self.base_estimator,
```

Comment:
Can we pass a cloned the estimator here? I mean `clone(self.base_estimator)` or we clone all of them beforehand `self.estimators_ = [clone(self.base_estimator) for _ in range(self.n_estimators)]` and then index into them `self.estimators_[i]` in the loop

Reply:
Let's follow your suggestion👍 , it looks like cloning all trees before the parallelization is also the convention in scikit-learn [Here](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/_forest.py#L377).